proto/frequenz/api/common/v1/grid/delivery_duration.proto

```diff
@@ -0,0 +1,59 @@
+// Frequenz definitions of the time increment used for electricity 
+// deliveries and trading.
+//
+// Copyright 2023 Frequenz Energy-as-a-Service GmbH
+//
+// Licensed under the MIT License (the "License");
+// you may not use this file except in compliance with the License.
+
+syntax = "proto3";
+
+package frequenz.api.common.v1.grid;
```

Comment:
Is this import being used anywhere?

Reply:
Ah sorry, please ignore this comment. I somehow misread `package` as `import` :face_with_spiral_eyes: